lamindb/_record.py

```diff
@@ -224,7 +224,9 @@ def __init__(record: Record, *args, **kwargs):
                 message = _format_django_validation_error(record, e)
                 raise FieldValidationError(message) from e
     elif len(args) != len(record._meta.concrete_fields):
-        raise ValueError("please provide keyword arguments, not plain arguments")
+        raise FieldValidationError(
+            f"Use keyword arguments instead of positional arguments, e.g.: {record.__class__.__name__}(name='...'). See model documentation for valid fields."
```

Comment:
> See model documentation for valid fields.

Simply remove. There are no "models" in lamindb (it'd be called registry) but it's also evident. And typically they should just switch on intellisense rather than looking into the docs.

Reply:
Done.